integration/testnet/network.go

```diff
@@ -216,7 +218,7 @@ func (net *FlowNetwork) ConsensusFollowerByID(id flow.Identifier) consensus_foll
 // Otherwise fails the test.
 func (net *FlowNetwork) ContainerByName(name string) *Container {
 	container, exists := net.Containers[name]
-	require.True(net.t, exists)
+	require.True(net.t, exists, fmt.Sprintf("container %s does not exists", name))
```

Comment:
```suggestion
	require.True(net.t, exists, "container %s does not exists", name)
```

The assertion functions will actually handle formatting